packages/bitcore-wallet-client/src/lib/payproV2.ts

```diff
@@ -72,6 +72,8 @@ export class PayProV2 {
       if (requestOptions.args) {
         if (requestOptions.method == 'post' || requestOptions.method == 'put') {
           r.send(requestOptions.args);
+        } else if (requestOptions.method === 'get') {
```

Comment:
What if the timeout occurs on one of the posts?

Reply:
Addressed the post, by also adding a 10 second timeout. An action sheet will be thrown on copay's side when caught.